examples/dev_flow/debug/debugging_visual.rst

```diff
@@ -0,0 +1,72 @@
+
+.. _examples_dev_flow_debug_visual:
+
+
+Debugging shared libraries with Visual Studio
+=============================================
+
+Conan packages don't contain the necessary information for debugging libraries with Visual Studio by default.
+This information is stored in PDBs that are generated during the compilation of the libraries. When using Conan these
+PDBs are generated in the build folder, which isn't always available, making impossible to debug the dependencies.
+
+The goal is to have the necessary PDB files in the package folder, so that Visual is able to debug through the
+dependencies of a project. To solve this we created a hook that copies the PDBs created in the build folder to the
+package folder. This behavior can't be forced by default because PDB files are usually larger than the whole package,
+and it would greatly increase the package sizes.
```

Comment:
You can continue here saying that to prevent that situation, you can store the pdb's in the package folder, so even after a `conan cache clean` you will be able to debug the dependencies.